tests/test_constrained.py

```diff
@@ -489,9 +519,17 @@ def test_error_advanced(self):
         self.assertAlmostEqual(const_result.value(b), std_result.value(b), 4)
         self.assertAlmostEqual(const_result.value(c), std_result.value(c), 4)
 
-        self.assertAlmostEqual(const_result.stdev(a), std_result.stdev(a), 4)
-        self.assertAlmostEqual(const_result.stdev(b), std_result.stdev(b), 4)
-        self.assertAlmostEqual(const_result.stdev(c), std_result.stdev(c), 4)
+        # This used to be a tighter equality test, but since we now use the
+        # Hessian we actually get a more accurate value from the standard fit
+        # then for MINPACK. Hence we check if it is roughly equal, and if our
+        # stdev is greater than that of minpack.
+        self.assertAlmostEqual(const_result.stdev(a) / std_result.stdev(a), 1, 2)
+        self.assertAlmostEqual(const_result.stdev(b) / std_result.stdev(b), 1, 1)
+        self.assertAlmostEqual(const_result.stdev(c) / std_result.stdev(c), 1, 2)
+
+        self.assertGreaterEqual(const_result.stdev(a), std_result.stdev(a))
+        self.assertGreaterEqual(const_result.stdev(b), std_result.stdev(b))
+        self.assertGreaterEqual(const_result.stdev(c), std_result.stdev(c))
```

Comment:
Does this mean MINPACK *under*estimates covariances?

Reply:
In this particular example, yes. But actually I should be a bit careful with my statements here, because I don't think it has to be a greater than in general, just that it is not the same.